keyboards/kingly_keys/little_foot/little_foot.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define SPLITSPACEBASE( \
```

Comment:
I think this should be:

```suggestion
#define LAYOUT_split_space_base( \
```
?